README.md

```diff
@@ -1,13 +1,57 @@
 # Northcoders News API
 
--- Install npm postgres in order to for the connection.js file to establish the connection pool.
+-- Link to hosted db 
+https://nc-news-project-8so2.onrender.com/
 
--- Install npm dotenv, the ENV variable references the environment variable process.env.NODE_ENV.
+-- This project is an API which allows the user to access the Northcoders News database. The API facilitates the sending of GET, POST, PATCH and DELETE requests by the user to said database.
 
-Set up a .env.test and a .env.development file in your directory and add it to the git ignore to protect it.
 
-This way you can also run the seed with test data via jest or you can run the seed with dev data via insomnia, postman etc.
 
--- Install npm express for the app file
+-- Clone this git repo from:
 
--- Install jest supertest, jest-sorted to run the seed.test.js file
\ No newline at end of file
+https://github.com/JJamC/nc-news-project.git
+
+
+
+-- Dependencies to install
+
+$ npm i postgres
+
+Install npm postgres in order for the connection.js file to establish the connection pool.
+
+$ npm i pg-format
```

Comment:
running npm install will do all of this for the user as these packages are detailed in your package json already so the user doesn tneed to install them manually